src/Assets/CRUD.js

```diff
@@ -0,0 +1,66 @@
+/*eslint-disable*/
+export  class CRUD {
+  static createTask() {
+    const inputData = document.getElementById('inputText').value;
+    console.log(inputData);
+    const saveData = document.getElementById('formText');
+    const storage = this.getAllTasks().length;
+    const task = [{ description: inputData, completed: false, index: storage }];
+    this.saveTask(task);
+    this.showTask();
+    saveData.reset();
+  }
+
+  static showTask() {
+    const div = document.querySelector('#todo-container');
+    div.innerHTML = '';
+    this.getAllTasks().forEach((task) => {
+      let checked = '';
+      let style = '';
+      let styleDescription = '';
+      const styleText='style="display:none"';
+      if (task.completed) {
+        styleDescription = 'style="text-decoration: line-through"';
+        checked = 'checked';
+        style = 'class="line"';
+      }
+      div.innerHTML += `<li><input class="checkbox-class" type="checkbox" id="checkbox-${task.index}" ${checked} /> <h3  id="d${task.index}" ${style} ${styleDescription} >${task.description}</h3><input class="edit-class" type="text" value="${task.description}" id="edit-${task.index}" ${styleText}/><button type="button" id="editBttn-${task.index}" class="editButton"></button><button type="button" class="deleteButton" id="delete-${task.index}"></button></li>`;
+    });
+  }
+
+  static removeTask(index) {
+    if (index > -1) {
+      const newData = this.getAllTasks();
+      newData.splice(index, 1);
+      for (let i = index; i < this.getAllTasks().length - 1; i += 1) {
+        newData[i].index = String(i, 10);
+      }
+      this.updateTask(newData);
+      this.showTask();
+    }
+  }
```

Comment:
Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
<img width="523" alt="Screenshot 2021-08-21 at 22 24 28" src="https://user-images.githubusercontent.com/34798898/130335160-87ae4b9b-4cb3-4b20-a611-db5b262591ca.png">

Reply:
![image](https://user-images.githubusercontent.com/78378168/130335397-8024181a-7a60-4a15-a28f-07e0c74b5d4b.png)
Indexes also working well

Reply:
I cannot understand why are those things happening in your case but when I checked everything in my browser it is working properly.